WEB-INF/classes/gov/noaa/pfel/erddap/util/EDStatic.java

```diff
@@ -1933,7 +1934,8 @@ public static int convertToPublicSourceUrlFromSlashPo(String tFrom) {
       emailFromAddress = getSetupEVString(setup, ev, "emailFromAddress", (String) null);
       emailEverythingToCsv = getSetupEVString(setup, ev, "emailEverythingTo", ""); // won't be null
       emailDailyReportToCsv =
-          getSetupEVString(setup, ev, "emailDailyReportTo", ""); // won't be null
+          Optional.ofNullable(getSetupEVString(setup, ev, "emailDailyReportTo", ""))
```

Comment:
Because this defaults to a string (""), if there is no value for emailDailyReportTo in setup.xml this will just use "".

I believe this should work:
```
emailDailyReportToCsv =
          Optional.ofNullable(getSetupEVString(setup, ev, "emailDailyReportTo", (String) null))
              .orElse(getSetupEVString(setup, ev, "emailDailyReportsTo", ""));
```

Reply:
Thanks for pointing that out